ui/src/shared/components/cells/Cells.tsx

```diff
@@ -1,5 +1,6 @@
 // Libraries
 import React, {Component} from 'react'
+import {connect} from 'react-redux'
```

Comment:
Since we don’t seem to be using redux, we can remove the connect here

Reply:
got it